source/geometries/NextDemoSiPMBoard.cc

```diff
@@ -208,25 +211,59 @@ void NextDemoSiPMBoard::Construct()
 
 
   /// Mask Holes
+  G4String coating_name = "BOARD_COATING";
+  G4Material* tpb = MaterialsList::TPB();
+  tpb->SetMaterialPropertiesTable(OpticalMaterialProperties::TPB());
+  G4OpticalSurface* coating_opsurf =
+    new G4OpticalSurface(coating_name + "_OPSURF", glisur, ground,
+                         dielectric_dielectric, .01);
+
   G4VSolid* hole_solid = nullptr;
   G4String hole_name = "BOARD_MASK_HOLE";
   if (hole_type_ == "rounded"){
     hole_solid = new G4Tubs(hole_name, 0., hole_diam_/2., mask_thickn_/2., 0, 360.*deg);}
   else if (hole_type_ == "rectangular"){
     hole_solid = new G4Box(hole_name, hole_x_/2., hole_y_/2., mask_thickn_/2.);}
 
-  G4LogicalVolume* hole_logic =
-    new G4LogicalVolume(hole_solid, mother_gas, hole_name);
+  G4LogicalVolume* hole_logic = new G4LogicalVolume(hole_solid, mother_gas, hole_name);
 
   // Generate SiPM positions
   GenerateSiPMPositions();
 
   G4double sipm_posz = - mask_thickn_/2. + sipm_z_dim/2.;
 
-  new G4PVPlacement(sipm_rot, G4ThreeVector(0., 0., sipm_posz), sipm_->GetLogicalVolume(),
-                    sipm_->GetLogicalVolume()->GetName(), hole_logic,
-                    false, 0, false);
+  G4VPhysicalVolume* hole_phys_ = new G4PVPlacement(sipm_rot, G4ThreeVector(0., 0., sipm_posz), sipm_->GetLogicalVolume(),
```

Comment:
This placement is that of the SiPM inside the hole, right? If so it should be called `sipm_phys`, otherwise it's misleading.